scvi/hub/hub_metadata.py

```diff
@@ -0,0 +1,188 @@
+import json
+from dataclasses import dataclass
+from typing import List, Optional
+
+import torch
+from dataclasses_json import dataclass_json
+from huggingface_hub import ModelCard, ModelCardData
+
+from ._constants import (
+    ANNDATA_VERSION_TAG,
+    ANNOTATED_TAG,
+    DEFAULT_MISSING_FIELD,
+    DEFAULT_NA_FIELD,
+    DEFAULT_PARENT_MODULE,
+    HF_LIBRARY_NAME,
+    MODALITY_TAG,
+    MODEL_CLS_NAME_TAG,
+    SCVI_VERSION_TAG,
+    TISSUE_TAG,
+)
+from ._utils import _get_counts_and_latent_info
+from .model_card_template import template
+
+
+@dataclass_json
+@dataclass
+class HubMetadata:
+    """Placeholder docstring. TODO complete."""
+
+    data_cell_count: int
+    data_gene_count: int
```

Comment:
see note above about generalizing cell and gene count. This also breaks in multimodal situations. I prefer using the summary statistics block from the model

Reply:
note to self: remove `model_setup_anndata_args` and use `view_setup_anndata` instead. keep `model_init_params`